src/main/java/pl/edu/agh/mwo/reporter/Main.java

```diff
@@ -12,13 +12,17 @@
 import pl.edu.agh.mwo.reporter.report.printer.Report1Printer;
 import pl.edu.agh.mwo.reporter.report.printer.Report2Printer;
 
+import java.io.File;
 import java.io.IOException;
 import java.nio.file.Path;
 import java.util.ArrayList;
 
 public class Main {
 
-    final static String OUTPUT_PATH = "resources\\Reports.xlsm";
+    //final static String OUTPUT_PATH = "resources\\Reports.xlsx"; //win
+   // final static String OUTPUT_PATH = "resources/Reports.xlsx"; //mac
+    final static String OUTPUT_FILE = "Reports.xls";
+    final static String OUTPUT_PATH = "resources" + File.separator + OUTPUT_FILE; //mac i ?win
```

Comment:
Zostaw jedną działającą ścieżke a reszte usuń. I tak docelowo to będzie wczytywane z parametru programu

Reply:
done